explorer/interpreter/resolve_names.cpp

```diff
@@ -82,6 +82,13 @@ static auto AddExposedNames(const Declaration& declaration,
       }
       break;
     }
+    case DeclarationKind::InterfaceExtendsDeclaration: {
+      // TODO: Should we add names from the extended constraint?
+      break;
+    }
+    case DeclarationKind::InterfaceImplDeclaration: {
+      break;
+    }
```

Comment:
Would it make sense to group these, ImplDeclaration, and MixDeclaration together? If they're all not providing names for resolution, perhaps they can all clearly be the same?